components/src/dynamo/planner/tests/unit/test_aic_interpolation.py

```
@@ -140,7 +140,7 @@ def test_dep(self):
         self._assert_identity(kw)
 
     def test_hybrid_tep_plus_dp(self):
-        # Hybrid: attention TP=2 × DP=4, MoE TP=2 × EP=4, 16 total GPUs.
+        # Hybrid: attention TP=2 × DP=4, MoE TP=2 × EP=4, 8 physical GPUs.
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)